garbage-collect/handler.go

```diff
@@ -46,6 +47,15 @@ func Handle(req []byte) string {
 		}
 	}
 
+	auditEvent := sdk.AuditEvent{
```

Comment:
Please do a rebase with the latest changes on this function. The garbage collection should post useful information such as the amount of deleted functions. See my latest update.